lib/go/test/flow_dkg_test.go

```diff
@@ -63,27 +66,27 @@ func TestDKG(t *testing.T) {
 		)
 	})
 
-	t.Run("Should not be able to register a voter if the node hasn't been registered as a consensus node", func(t *testing.T) {
+	t.Run("Should be able to register a voter before the dkg is enabled", func(t *testing.T) {
 
-		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateCreateDKGParticipantScript(env), DKGAddress)
+		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateCreateDKGParticipantScript(env), jordanAddress)
 
 		_ = tx.AddArgument(cadence.NewAddress(DKGAddress))
-		_ = tx.AddArgument(cadence.NewString(adminID))
+		_ = tx.AddArgument(cadence.NewString(accessID))
```

Comment:
just checking this wasn't a bad find+replace -- not seeing where the variable definition changed

Reply:
Since the registration succeeds now, I have to use a different ID than before because the registration later that uses it would fail if I didn't